pkg/handler/employee/employee.go

```diff
@@ -81,6 +83,107 @@ func (h *handler) One(c *gin.Context) {
 	c.JSON(http.StatusOK, view.CreateResponse[any](view.ToEmployeeListData(employee), nil, nil, nil))
 }
 
+// Update godoc
+// @Summary Edit employee by id
+// @Description Edit employee by id
+// @Tags Employee
+// @Accept  json
+// @Produce  json
+// @Param id path string true "Employee ID"
+// @Param fullName body string true "fullName" maxlength(99)
+// @Param email body string true "email"
+// @Param phone body string true "phone" minlength(10)  maxlength(12)
+// @Param lineManager body model.UUID true "lineManager"
+// @Param discordID body string true "discordID"
+// @Param githubID body string true "githubID"
+// @Param role body model.UUID true "role"
+// @Param chapter body model.UUID true "chapter"
+// @Param seniority body model.UUID  true "seniority"
+// @Param stack body []model.UUID true "stack"
+// @Param DoB body time.Time true "DoB"
+// @Param gender body string true "gender"
+// @Param address body string true "address" maxlength(200)
+// @Param personalEmail body string true "personalEmail"
+// @Param Authorization header string true "jwt token"
+// @Success 200 {object} view.EditEmployeeResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /employees/{id}/edit [post]
+func (h *handler) Update(c *gin.Context) {
+	// 1. parse id from uri, validate id
+	var params struct {
+		ID string `uri:"id" binding:"required"`
+	}
+
+	if err := c.ShouldBindUri(&params); err != nil {
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, err, params))
+		return
+	}
+
+	// 2. parse json body from request
+	type EditEmployeeRequest struct {
```

Comment:
move to `/pkg/handler/employee/request.go`

Reply:
cái này em chưa biết nên sao, em hỏi anh @namnhce thì anh kêu để ở bên này á anh

Reply:
move luôn nha em. để trong file request.go nhé

Reply:
ok anh